frontend/micro-ui/web/micro-ui-internals/packages/modules/engagement/src/pages/employee/CitizenSurveys/SurveyDetails.js

```
@@ -194,12 +194,40 @@ const SurveyDetails = ({ location, match }) => {
+  
+    const updatedSurveyData = {
+      ...surveyData,
+      startDate: toEpoch(surveyData.fromDate, surveyData.fromTime),
+      endDate: toEpoch(surveyData.toDate, surveyData.toTime),
+      tenantId,
+      questions: surveyData.questions.map(filterQuestion),
+      status: "INACTIVE"
+    };
+  
+    // Remove fromDate and toDate
+    delete updatedSurveyData.fromDate;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)